edgelet/iotedge/src/main.rs

```diff
@@ -388,6 +422,37 @@ fn run() -> Result<(), Error> {
                 std::process::exit(1);
             }
         },
+        ("system", Some(args)) => match args.subcommand() {
+            ("logs", Some(args)) => {
+                let jctl_args: Vec<&OsStr> = args
+                    .values_of_os("args")
+                    .map_or_else(Vec::new, std::iter::Iterator::collect);
+
+                System::get_system_logs(&jctl_args)
+            }
+            ("restart", Some(_args)) => System::system_restart(),
+            ("status", Some(_args)) => System::get_system_status(),
+            ("set-log-level", Some(args)) => {
+                if let Some(log_level) = args.value_of("log_level") {
+                    if let Ok(level) = log::Level::from_str(log_level) {
+                        System::set_log_level(level)
+                    } else {
+                        eprintln!(
+                            r#"Error: log_level must be one of "trace", "debug", "info", "warn", or "error""#
+                        );
+                        std::process::exit(1);
+                    }
+                } else {
+                    eprintln!("Error: missing value for log_level");
```

Comment:
instead of doing this error handling manually, just pass `.required()` to clap when defining the `log_level` flag.